File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int num: nums){
            if(even(num)){
                count++;
            }
        }
        return count;        
    }
    
    int digits(int num){
        if(num < 0){
            num = num * -1;
        }
        return (int)(Math.log10(num)) + 1;
    }
    
    boolean even(int num){
        int numberOfDigits = digits(num);
        return numberOfDigits % 2 == 0;
    }
}

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let count = 0;
    nums.map((ele)=>{
        if(Math.abs(ele).toString().length % 2 === 0){
            count++;
        }
    })
    return count;
};